flink-end-to-end-tests/flink-tpcds-test/tpcds-tool/README.md

```diff
@@ -0,0 +1,28 @@
+# TPC-DS tool
+
+This directory contains the necessary scripts/files to run a TPC-DS test.
+
+## data_generator.sh scripts
+
+Scripts that to download the TPC-DS data generator and generate test data, the supported OS including GNU/Linux OS and Mac OS.
```

Comment:
data_generator.sh <generator_dir> <scale_factor> < data_dir> <retry_scripts_dir>

<generator_dir>, the directory to save TPC-DS data generator.
....